server/api/register_handler.py

```diff
@@ -1,5 +1,5 @@
 import json
-from flask import jsonify, request, Blueprint, Response
+from flask import jsonify, request, Blueprint, Response, current_app
```

Comment:
Why do we need current_app here as well?

Reply:
Also from testing, removing